scim.sgnl.yaml

```
@@ -0,0 +1,457 @@
+      - name: profileUrl
+        externalId: profileUrl    
+        type: String
+      - name: userType
+        externalId: userType
+        type: String  
+      - name: title
+        externalId: title
+        type: String
+      - name: preferredLanguage
+        externalId: preferredLanguage
+        type: String
+      - name: locale  
+        externalId: locale
+        type: String
+      - name: timezone
+        externalId: timezone
+        type: String
+      - name: active
+        externalId: active
+        type: Bool 
+      
+      # Enterprise User Extension
+      - name: employeeNumber
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].employeeNumber
+        type: String
+        indexed: true
+      - name: costCenter
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].costCenter
+        type: String
+      - name: organization
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].organization
+        type: String
+      - name: division
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].division
+        type: String
+      - name: department
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].department
+        type: String
+      - name: managerId
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
+        type: String  
+        indexed: true
+      - name: managerDisplayName
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.displayName
+        type: String
+      
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version 
+        externalId: $.meta.version
+        type: String  
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  Group:
+    displayName: SCIMGroup
+    externalId: Groups
+    description: Group entity in SCIM
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  GroupMember:
+    parent: Users  # External ID of the parent entity
+    displayName: SCIMGroupMember
+    externalId: groups
+    description: Entity representing SCIM user group members
```

That description seems incorrect.
```suggestion
    description: Entity representing the SCIM groups each user belongs to.
```